src/writeData/containers/TelegrafPluginsPage.tsx

```diff
@@ -22,21 +22,31 @@ import placeholderLogo from 'src/writeData/graphics/placeholderLogo.svg'
 
 // Utils
 import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
+import {event, normalizeEventName} from 'src/cloud/utils/reporting'
 
 // Styles
 import 'src/writeData/components/WriteDataDetailsView.scss'
 import {MarkdownRenderer} from 'src/shared/components/views/MarkdownRenderer'
 
-const codeRenderer: Renderer<HTMLPreElement> = (props: any): any => (
-  <CodeSnippet text={props.value} label={props.language} />
-)
-
 const TelegrafPluginsPage: FC = () => {
   const {contentID} = useParams()
   const {name, markdown, image} = WRITE_DATA_TELEGRAF_PLUGINS.find(
     item => item.id === contentID
   )
 
+  const eventName = normalizeEventName(name)
+  useEffect(() => {
+    event('telegraf_tile.' + eventName + '.config_viewed')
```

Comment:
```suggestion
    event(`telegraf_tile.${eventName}.config_viewed`)
```
prefer string interpolation